docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2168,6 +2168,18 @@ If the remote HTTP image renderer service runs on a different server than the Gr
 Concurrent render request limit affects when the /render HTTP endpoint is used. Rendering many images at the same time can overload the server,
 which this setting can help protect against by only allowing a certain number of concurrent requests. Default is `30`.
 
+### default_image_width
```

Comment:
@chri2547 Could you take a look before we merge, please?

Reply:
@AgnesToulet , on it! Thank you.